system-setup.py

```diff
@@ -22,23 +22,35 @@ def common_first(self):
 
     def debian_compat(self):
         self.install("build-essential")
+        self.install("python3-psutil")
 
     def redhat_compat(self):
         self.group_install("'Development Tools'")
         self.install("redhat-lsb-core")
+        if not self.dist == "amzn":
+            self.install("epel-release")
+            self.install("python3-devel libaec-devel")
+            self.install("python36-psutil")
+        else:
+            self.run("amazon-linux-extras install epel", output_on_error=True)
+            self.install("python3-devel")
+            self.pip_install("psutil")
 
     def fedora(self):
         self.group_install("'Development Tools'")
-
-    def macos(self):
-        if sh('xcode-select -p') == '':
-            fatal("Xcode tools are not installed. Please run xcode-select --install.")
+        self.install("python3 python3-psutil python3-networkx")
 
     def common_last(self):
+        self.run("python3 -m pip uninstall -y RLTest || true")
         self.install("lcov")
         if not self.has_command("ramp"):
             self.pip_install("git+https://github.com/RedisLabs/RAMP@master")
-        self.pip_install("-r tests/requirements.txt")
+        if not self.has_command("RLTest"):  
+            # redis-py-cluster should be installed from git due to redis-py dependency
+            self.pip_install("--no-cache-dir git+https://github.com/Grokzen/redis-py-cluster.git@master")
+            self.pip_install("--no-cache-dir git+https://github.com/RedisLabsModules/RLTest.git@master")
```

Comment:
its better to use a static version instead of master to make sure we don't break due to a new RLTest version

Reply:
Also, im not sure this is needed as we use `install -r ` in line #53

Reply:
fixed as per your comment :+1: